microsoft/testsuites/acctests/sgxtest.py

```diff
@@ -0,0 +1,217 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import os
+import pytest
+import subprocess
+
+from typing import Any
+from pathlib import Path
+from typing import Any
+from assertpy import assert_that
+
+from lisa import (
+    CustomScript,
+    CustomScriptBuilder,
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+from lisa.operating_system import Posix
+from lisa.tools import Echo, Uname
+
+########### HELPER METHODS ###################
+
+@staticmethod
+def getOEInstalledPath():
+    return os.path.expanduser("~/openenclave-install/")
+
+@staticmethod
+def getSGXVersion():
+    if isOeInstalled():
+        openenclave_install_path = getOEInstalledPath()
+        print("!!!!!!!!!!!!!!!!!!!!!!!!!!STARTED")
+        print(openenclave_install_path)
```

Comment:
Add some words to say what's the path in the log. When it combines with other logs, it's hard to know what it is, where it's from.